site/components/src/test/java/uk/nhs/hee/web/uk/nhs/hee/web/components/BreadcrumbComponentTest.java

```diff
@@ -0,0 +1,152 @@
+package uk.nhs.hee.web.uk.nhs.hee.web.components;
+
+import org.hippoecm.hst.configuration.hosting.Mount;
+import org.hippoecm.hst.configuration.site.HstSite;
+import org.hippoecm.hst.configuration.sitemap.HstSiteMap;
+import org.hippoecm.hst.configuration.sitemap.HstSiteMapItem;
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import org.hippoecm.hst.core.container.ComponentManager;
+import org.hippoecm.hst.core.linking.HstLink;
+import org.hippoecm.hst.core.linking.HstLinkCreator;
+import org.hippoecm.hst.core.request.HstRequestContext;
+import org.hippoecm.hst.core.request.ResolvedMount;
+import org.hippoecm.hst.core.request.ResolvedSiteMapItem;
+import org.hippoecm.hst.mock.core.component.MockHstRequest;
+import org.hippoecm.hst.site.HstServices;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.onehippo.cms7.essentials.components.ext.DoBeforeRenderExtension;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import uk.nhs.hee.web.components.BreadcrumbComponent;
+import uk.nhs.hee.web.components.beans.BreadcrumbLinkTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.mockito.Mockito.*;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*", "javax.script.*"})
+@PrepareForTest({HstServices.class, RequestContextProvider.class})
+public class BreadcrumbComponentTest {
```

Comment:
@smanivasag May be this library can help us on HST testing a bit instead of mock everything ourselves 
https://github.com/bloomreach-forge/brut ?

Reply:
Sure, thanks @thojava. Will take a look.

Reply:
@thojava  @duncanheron - Since, this comment is about changing the unit testing strategy which can be done later, how about the PR can be approved so that it can be deployed to brCloud ? Thanks!

Reply:
@smanivasag  Agree - if we arre in agreement we should change the test to make use of brut, we can create a new backlog ticket

Reply:
@duncanheron - Happy for you to create a separate ticket on using brut for unit testing